examples/resources/biganimal_fa_replica/aws/resource.tf

```diff
@@ -0,0 +1,66 @@
+terraform {
+  required_providers {
+    biganimal = {
+      source  = "EnterpriseDB/biganimal"
+      version = "0.3.0"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "3.3.1"
+    }
+  }
+}
+
+variable "cluster_name" {
+  type        = string
+  description = "The name of the faraway replica cluster."
+}
+
+variable "source_cluster_id" {
+  type        = string
+  description = "BigAnimal source cluster ID"
+}
+
+variable "project_id" {
+  type        = string
+  description = "BigAnimal Project ID"
+}
+
+resource "biganimal_fareplica" "faraway_replica" {
```

Comment:
I think we should go with the explicit name of `biganimal_faraway_replica` as the resource name instead of `biganimal_fareplica`. WDYT? 

Also, the example directory name should be equivalent to the resource. For example, a `biganimal_faraway_replica` resource should have an example under `examples/resources/biganimal_faraway_replica/resource.tf` so that the [tfplugindocs](https://github.com/hashicorp/terraform-plugin-docs) can add the code snippet to the corresponding documentation. 

The official documentation docs are [here](https://developer.hashicorp.com/terraform/registry/providers/docs).

Reply:
Sounds good, fixed.